lib/client.js

```diff
@@ -26,16 +26,18 @@ Client.prototype.connect = function connect (handler, callback) {
     client._ws._socket.unref();
     client._heartbeat.unref();
 
+    const info = {
+      pid: process.pid
```

Comment:
I don't think we can use the process id. Since the server can be connected to by multiple hosts, the process ids could collide across machines. Really unlikely, but possible.